src/index.d.ts

```diff
@@ -0,0 +1,73 @@
+declare module "react-dropzone-s3-uploader" {
```

Comment:
This file was pulled from `exec_online`:`components/enterprise_portal/client/types/react-dropzone-s3-uploader.d.ts`